src/PlacesAutocomplete.js

```diff
@@ -346,6 +350,7 @@ PlacesAutocomplete.defaultProps = {
   onError: (status) => console.error('[react-places-autocomplete]: error happened when fetching data from Google Maps API.\nPlease check the docs here (https://developers.google.com/maps/documentation/javascript/places#place_details_responses)\nStatus: ', status),
   classNames: {},
   autocompleteItem: ({ suggestion }) => (<div>{suggestion}</div>),
+  filterPredictions: null,
```

Comment:
@pioul Let's add a more meaningful default

```js
filterPredictions: (prediction) => true
```

So that we can get rid of the null check in the code 👍